### PR TITLE
Upgrade to CakeBuild v3

### LINKED ISSUE #17: Upgrade to CakeBuild V3

### PR DESCRIPTION
**Pull request by Tanzy**
Upgrade to the latest version of Cake

## Description
Updated to CakeBuild v3

## Related Issue
#17

### CODE REVIEW

File: recipe.cake [hunk 1]
```diff
@@ -1,22 +1,23 @@
-#load nuget:?package=Cake.Recipe&version=1.0.0
+#load nuget:?package=Cake.Recipe&version=3.0.1
 
 Environment.SetVariableNames();
 
 BuildParameters.SetParameters(context: Context,
                             buildSystem: BuildSystem,
                             sourceDirectoryPath: "./src",
-                            title: "Cake.Grunt",
+                            title: "Cake.Gulp",
```
Comment (nils-a) [inline recipe.cake]: @Tanzy are you sure, this change is correct? 
Renaming the `title` to `Cake.Gulp` will result in strange behaviors here, and maybe also in Cake.Gulp.

Comment (Tanzy) [reply recipe.cake]: good spot on this ta, changed

**Comment by Tanzy** [comment]
This is getting too many issues with the build script so unfortunately it needs to be shelved.

**Comment by nils-a** [comment]
@Tanzy Sorry, I pushed into your branch. So sorry for doing this without asking!!

**Comment by Tanzy** [comment]
@nils-a no worries, I’m afraid that I don’t know what’s wrong or needs doing to this now. So please, push away :D

**Comment by nils-a** [comment]
@Tanzy this felt a bit like stumbling in the dark 😁 You should probably squash-merge all the tries here.

Given the time, you should consider re-enabling CakeContribute.Guidelines. I know the person who built it, and it's quite good. 😉

**Comment by Tanzy** [comment]
@nils-a yeh, tried to put the CakeContribute.Guidelines in but caused issues. May have been because I’m on an M1 Mac and can’t remember now. I will look at getting it in again before I merge this, do it in one. 

Then I have to remember how to release this 😄

**Comment by nils-a** [comment]
Maybe https://cake-contrib.github.io/Cake.Recipe/docs/usage/creating-release can help you there. 😉

**Comment by Tanzy** [comment]
Ta

**Pull request merged**